tests/models/deta/test_modeling_deta.py

```diff
@@ -58,13 +58,15 @@ def __init__(
         attention_probs_dropout_prob=0.1,
         num_queries=12,
         num_channels=3,
-        image_size=196,
+        image_size=224,
```

Comment:
This is related to CI.
Small image size will cause error like this. https://github.com/IDEA-Research/GroundingDINO/issues/181